README.md

```diff
@@ -153,10 +160,14 @@ which can be executed in the same way.
 The same thing can also be accomplished using `irmin`, the command-line
 application installed with `irmin-unix`, by running:
 
-    $ echo "root: ." > irmin.yml
-    $ irmin init
-    $ irmin set foo/bar "testing 123"
-    $ irmin get foo/bar
+```bash
+$ echo "root: ." > irmin.yml
+$ irmin init
+$ irmin set foo/bar "testing 123"
+irmin: [WARNING] Reference refs/heads/main not found.
```

Comment:
This will be fixed by https://github.com/mirage/irmin/pull/1698, which should be merged first. (But this PR can still be reviewed separately.)